lib/orphans.nix

```diff
@@ -0,0 +1,75 @@
+{ lib }:
+let
+  inherit (lib)
+    types
+    optional
+    isDerivation
+    attrByPath
+  ;
+  inherit (lib.lists)
+    flatten
+    any
+  ;
+  inherit (lib.asserts)
+    assertMsg
+  ;
+  inherit (builtins)
+    length
+    concatStringsSep
+    mapAttrs
+    attrValues
+    tryEval
+  ;
+  isAttr = types.attrs.check;
+in
+rec {
+  /* Returns a list of maintainers from a package, empty list if inacessible
+  */
```

Comment:
Please follow the way it is documented in other `lib` files: https://github.com/NixOS/nixpkgs/blob/master/lib/attrsets.nix#L15-L23

The "Examples" part is specially useful to have a quick look on how this is supposed to work. Just avoid using any real nicknames, instead use something like "foo" as replacements.